Macros/CHAS_summary_vars.sas

```diff
@@ -0,0 +1,964 @@
+			renter_3plusbr_aff50_&years. = "Renter-occupied 3+ bedroom housing units affordable at 50% AMI, &years_dash."
+			renter_allbr_aff50_&years. = "Renter-occupied housing units affordable at 50% AMI, &years_dash."
+			owner_01br_tot_&years. = "Owner-occupied 0-1 bedroom housing units, &years_dash."
+			owner_2br_tot_&years. = "Owner-occupied 2 bedroom housing units, &years_dash."
+			owner_3plusbr_tot_&years. = "Owner-occupied 3+ bedroom housing units, &years_dash."
+			owner_01br_aff50_&years. = "Owner-occupied 0-1 bedroom housing units affordable at 50% AMI, &years_dash."
+			owner_2br_aff50_&years. = "Owner-occupied 2 bedroom housing units affordable at 50% AMI, &years_dash."
+			owner_3plusbr_aff50_&years. = "Owner-occupied 3+ bedroom housing units affordable at 50% AMI, &years_dash."
+			owner_allbr_aff50_&years. = "Owner-occupied housing units affordable at 50% AMI, &years_dash."
+			;
+
+	%Pct_calc( var=Powner_units_tot, label=% Owner-occupied housing units, num=owner_units_tot, den=all_units_tot, years= &years. );
+	%Pct_calc( var=Pforsale_units_tot, label=% Vacant housing units for sale, num=forsale_units_tot, den=all_units_tot, years= &years. );
+	%Pct_calc( var=Prenter_unit_tot, label=% Renter-occupied housing units, num=renter_unit_tot, den=all_units_tot, years= &years. );
+	%Pct_calc( var=Pforrent_units_tot, label=% Vacant housing units for rent, num=forrent_units_tot, den=all_units_tot, years= &years. );
+
+	%Pct_calc( var=Prenter_unit_aff30, label=% Renter-occupied housing units affordable at 30% AMI, num=renter_unit_aff30, den=tot_aff30, years= &years. );
+	%Pct_calc( var=Pforrent_units_aff30, label=% For-rent housing units affordable at 30% AMI, num=forrent_units_aff30, den=tot_aff30, years= &years. );
+
+	%Pct_calc( var=Prenter_unit_aff50, label=% Renter-occupied housing units affordable at 50% AMI, num=renter_unit_aff50, den=tot_aff50, years= &years. );
+	%Pct_calc( var=Pforrent_units_aff50, label=% For-rent housing units affordable at 50% AMI, num=forrent_units_aff50, den=tot_aff50, years= &years. );
+
+	%Pct_calc( var=Powner_unit_aff50, label=% Owner-occupied housing units affordable at 50% AMI, num=owner_unit_aff50, den=tot_aff50, years= &years. );
+	%Pct_calc( var=Pforsale_units_aff50, label=% For-sale housing units affordable at 50% AMI, num=forsale_units_aff50, den=tot_aff50, years= &years. );
+
+	%Pct_calc( var=Prenter_01br_tot, label=% Renter-occupied 0-1 bedroom housing units, num=renter_01br_tot, den=renter_unit_tot, years= &years. );
+	%Pct_calc( var=Prenter_2br_tot, label=% Renter-occupied 2 bedroom housing units, num=renter_2br_tot, den=renter_unit_tot, years= &years. );
+	%Pct_calc( var=Prenter_3plusbr_tot, label=% Renter-occupied 3+ bedroom housing units, num=renter_3plusbr_tot, den=renter_unit_tot, years= &years. );
+
+	%Pct_calc( var=Prenter_01br_aff30, label=% Renter-occupied 0-1 bedroom housing units affordable at 30% AMI, num=renter_01br_aff30, den=renter_unit_aff30, years= &years. );
```

Comment:
I'm probably also going to want both row % and column % for these. 

so num = renter_01br_aff30 and denom = renter_01br_tot in addition to what you have.

Reply:
@rpitingolo i'm looking through the rest - but it doesn't look like this part was done?